README.md

```diff
@@ -87,6 +87,7 @@ You can see the bot in action at https://www.reddit.com/user/AnimalFactsBot/comm
 * jackal
 * jellyfish
 * kangaroo
+* kakapo
```

Comment:
This list has been capitalised now. If you can grab the latest repo and capitalise this that’ll sort it.